examples/docker-build/dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:10-stretch-slim as builder
```

Comment:
It would be a better idea to use a consistent runtime under the hood for all steps. Either Alpine or a Debian based one. There may be modules in `node_modules` with native bindings.